### PR TITLE
Dala 2497 heimathafen toolbox

### PR DESCRIPTION
**Pull request by emanueldfine**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
  - [x] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [ ] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] The version of main currently active on prod is deployed to a dev server with `Reset non-user related Docker Volumes & Re-populate` turned on
  - [ ] It's verified that the CD run is green
  - [ ] The data from prod is migrated using `./migrateData.sh dataland.com <SOURCE_API_KEY> <TARGET> <TARGET_API_KEY>` 
  - [ ] The feature branch is deployed to the same server with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the dev server
- [ ] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [ ] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [ ] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on dev server
- [ ] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/heimathafen/model/HeimathafenData.kt
Comment (aliabdoli77) [inline dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/heimathafen/model/HeimathafenData.kt:1]: for me: check after recategorization

File: dataland-frontend/src/components/forms/CreateHeimathafenDataset.vue
Comment (aliabdoli77) [inline dataland-frontend/src/components/forms/CreateHeimathafenDataset.vue:1]: In terms of structure and functions, this is almost identical to createEsgQuestionnairDataset.vue
Should a refactoring be considered here or do we keep createXdataset.vue always separate regardless?

Comment (emanueldfine) [reply dataland-frontend/src/components/forms/CreateHeimathafenDataset.vue:1]: Yes you are absolutely right, it is not very clean and nice that you need to create a component "CreateXXXDataset.vue" each time that you want to introduce an upload page for a framework.
This is why this ticket has been created: https://jira.d-fine.dev/browse/DALA-3386

When that ticket is finished, you won't need to create a new vue-component for each upload page.
For the view page for example, we already have a mechanism in place that avoids the necessity of creating a new vue-component for each framework.

However for now, we simply accpeted the fact, that we need to create the component `CreateHeimathafenDataset.vue`

**Comment by aliabdoli77** [comment]
Small Typo in DD
 
image

**Comment by aliabdoli77** [comment]
Is there any reason why these two fields have placeholders and no other field does?
image

**Comment by aliabdoli77** [comment]
Shouldn't the source here be a yes/no like other instances?
image

**Comment by aliabdoli77** [comment]
are we keeping the sub category name in the () for these two cases?
image

**Comment by aliabdoli77** [comment]
Has it been decided that we would keep the repeating subcategory names in ()?
image

**Comment by marcelrasbachdfine** [comment]
> are we keeping the sub category name in the () for these two cases? image

test

**Pull request merged**